### PR TITLE
VideoAnalytics: Updated Readme for Security Context

### PR DESCRIPTION
**Pull request by vivekrsintc**
This patch contains the changes of readme for setting
security context on helm charts enabling the cameras

Signed-off-by: Vivek R Soundararajan <vivek.rs@intel.com>

### CODE REVIEW

**Comment by vivekrsintc** [comment]
@vkb1 @iamvishwas

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 - [VideoAnalytics Module](#videoanalytics-module)
   - [Configuration](#configuration)
+  - [Set Securiry Context to Enable Basler/USB Camera or NCS2 devicen helm environment](#updating-security-context-of-videoingestion-helm-charts-for-enabling-k8s-environment-to-accessdetect-baslerusb-device)
```
Comment (vkb1) [inline README.md:5]: devicen typo. I see the VideoIngestion details in VideoAnalytics repo, something seems to be wrong here as we should be only talking about accelerators in VA.

File: README.md [hunk 2]
```diff
@@ -58,3 +60,29 @@ If `AppName` is `VideoAnalytics`, then the app's config would be fetched from
 
 One can use [JSON validator tool](https://www.jsonschemavalidator.net/) for
 validating the app configuration against the above schema.
+
+### Updating Security Context of VideoIngestion Helm Charts for enabling K8s environment to access/detect Basler/usb Device
```
Comment (vkb1) [inline README.md:64]: We shouldn't be mentioning about the VideoIngestion here.

File: README.md (hunk 1 above)
Comment (vkb1) [reply README.md:5]: If I remember correctly, we had the relevant readme info mentioned in both VI and VA readme for EII 3.0

Comment (vivekrsintc) [reply README.md:5]: yes, let me change it

**Comment by vkb1** [comment]
Hey @vivekrsintc ,
Please raise this MR in the inner source channel.

**Comment by vivekrsintc** [comment]
THis PR has been raised in innersource

**Pull request closed without merging**